WebServer/error_handler.py

```diff
@@ -0,0 +1,9 @@
+from read_page_inf import read_page
+
+
+def error_page(self, s_path):
+    response, page = read_page(s_path)
```

Comment:
why do you need to call `read_page()` here? You will return 404 even if `read_page` returns 200.
_Also `read_page` already returns status code_

Reply:
I wrote `read_page()` here to get _error.html_ page. Corrected a bit in the last commit.
Or maybe i don't fully understand what you mean.